npm/util/util.go

```diff
@@ -304,6 +304,10 @@ func GetIpSetFromLabelKV(k, v string) string {
 	return fmt.Sprintf("%s%s%s", k, IpsetLabelDelimter, v)
 }
 
+func IsKeyLabelSetName(k string) bool {
```

Comment:
This function is currently used only in v2, but tomorrow it can be used for v1 also, it is not tied to anything specific in v2, so keeping the name generic.